.github/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+<!--
+
+For more information on any of the below, please see our Contributing guidelines:
+https://github.com/FormidableLabs/react-fast-compare/blob/master/CONTRIBUTING.md#before-submitting-a-pr
+
+-->
+
+## Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context.
+
+## Checklist: (Feel free to delete this section upon completion)
+
+- [ ] All tests are passing
```

Comment:
Let’s add a bullet for performance has not significantly decreased for the benchmark